lib/modules/manager/bazelisk/extract.ts

```diff
@@ -0,0 +1,12 @@
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import type { PackageDependency, PackageFile } from '../types';
+
+export function extractPackageFile(content: string): PackageFile {
+  const dep: PackageDependency = {
+    depName: 'bazel',
+    currentValue: content.trim(),
+    datasource: GithubTagsDatasource.id,
```

Comment:
you should use `github-releases` datasource instead, as they publish GitHub releases.